favorite-movies.txt

```diff
@@ -1,2 +1,3 @@
-Borat: Cultural Learning for Make Benefit Great Nation of Kazakstan, Garfield,The Incredibles 2, The Room, 12 Years a Slave
+Borat: Cultural Learning for Make Benefit Great Nation of Kazakstan, Garfield,The Incredibles 2, The Room, 12 Years a Slave, It
```

Comment:
It is not a child friendly movie. Replace It with a child friendly movie. Not the other movies that aren't child friendly though. Only It.